graphql/connection_test.go

```diff
@@ -74,6 +74,19 @@ func TestConnection(t *testing.T) {
 			return []string{"can", "man", "cannot", "soban", "socan"}[i.Id-1]
 		},
 	})
+	inner.FieldFunc("innerConnectionWithSort", func() []Item {
+		return []Item{{1}, {2}, {3}, {4}, {5}}
+	}, schemabuilder.Paginated, schemabuilder.SortFields{
+		"numbers": func(ctx context.Context, i Item) int64 {
```

Comment:
Why do these need to be extra fields?  Can we build sorting on top of the existing field names?  Maybe just a whitelist of the "sortable" fields.

Reply:
Yes, but that's phase 3 of the plan.

Reply:
Maybe it's worth taking the time to do that now, given this work has taken less time than anticipated -- cc @stephen